docs/platform/visual/index.md

```diff
@@ -11,10 +11,9 @@ The Pentaho Visualization API provides a unified way to visualize data across th
 [Analyzer](http://www.pentaho.com/product/business-visualization-analytics), 
 [PDI](http://www.pentaho.com/product/data-integration), 
 [CDF](http://community.pentaho.com/ctools/cdf/)).
-Essentially, it is a set of abstractions that ensures total isolation between 
-applications, 
-visualizations and 
-configurations (the glue).
+
+Essentially, it is a set of abstractions that ensures isolation between
+applications, visualizations and configurations.
```

Comment:
Yes, I do not think that sentence is absolutely necessary in this file. The previous sentence states the purpose, which is enough for framing the user's mind.

That sentence can be safely moved to the article that provides more details about the viz api.